_docs/manage/policies.md

```diff
@@ -0,0 +1,118 @@
+
+```json
+{"spec":{"user": "bob", "namespace": "legal"}}
+{"spec":{"group": "compliance", "namespace": "legal"}}
+```
+
+The rules above will allow any user with the username bob, or in the group compliance, to access the legal namespace.
+
+## CLI
+
+### Adding policies
+Policies can be added via the CLI application using the `storageos policy create command. New policies can be supplied to this command with n-many positional arguments, a set of files or by reading from stdin. The positional arguments are expected to be in the form of either a single policy object or a json list of policy objects. The file input is expected to be in json line format (one object per line, the same as kubernetes). Reading from stdin will accept either json line format, or json objects/lists.
+
+#### Adding using positional args
+Policies can be added using positional arguments. They are expected to either be policy objects, or a json list of policy objects. In this example both are used.
+
+```bash
+$ storageos policy create '{"spec":{"user":"foo", "namespace": "*"}}' '[{"spec":{"user":"bar", "namespace": "testing"}}, {"spec":{"group":"baz", "namespace": "restricted"}}]'
+
+$ storageos policy ls
+ID                                     USER                GROUP               READONLY            APIGROUP            RESOURCE            NAMESPACE           NONRESOURCEPATH
+99b04c50-32a7-4869-8e23-1e6b8b887228   foo                                     false                                                       *
+c14aa3ff-99f7-d0fb-41e9-ad45b0919851   bar                                     false                                                       testing
+13de44a2-c50a-c225-9f65-871cfad0c838                       baz                 false                                                       restricted
+```
+
+#### Adding using jsonl files
+Policies can also be added using kubernetes style policy files (json line format). In this example we use these two files:
+
+p1.jsonl:
+```json
+{"spec":{"user":"foo", "namespace": "*"}}
+{"spec":{"user":"bar", "namespace": "testing"}}
+```
+
+p2.jsonl:
+```json
+{"spec":{"group":"baz", "namespace": "restricted"}}
+```
+
+These files are provided using the optional argument `--policies`
+```bash
+$ storageos policy create --policies='p1.jsonl,p2.jsonl'
+
+$ storageos policy ls
+ID                                     USER                GROUP               READONLY            APIGROUP            RESOURCE            NAMESPACE           NONRESOURCEPATH
+24e90cfe-9dbe-482c-3b51-8d2a9c0e57fd   foo                                     false                                                       *
+34c868ef-4e4b-fb71-f0f7-fa94e4652776   bar                                     false                                                       testing
+8ced2820-7ad1-bd46-4d9f-d7dab1ba4c67                       baz                 false                                                       restricted
+```
+#### Adding using stdin
```

Comment:
RE stdin in general: I think I have seen k8s commands that read from stdin to do stuff like this.

RE the --stdin flag: This is to prevent the command looking like it has hung. If it reads from stdin by default then a user that types `storageos policy create` may be tricked into thinking that the command has hung, being unaware that it is awaiting input.